### DISCUSSION TITLE
How to replace an autocmd?

### DISCUSSION THREAD

**Discussion by rochacbruno** [LazyVim/LazyVim#1408] [Q&A] [answered]
Hi, 

First, thanks for this amazing project.

I am trying to change one of the autocmds but having no effect.

lua/config/autocmds.lua
```lua
-- clear existing cmd
vim.api.nvim_create_augroup("lazyvim_highlight_yank", { clear = true })

-- register again
local yank_group = vim.api.nvim_create_augroup("custom_highlight_yank", {})
vim.api.nvim_create_autocmd("TextYankPost", {
  group = yank_group,
  pattern = "*",
  callback = function()
    vim.highlight.on_yank({
      higroup = "IncSearch",
      timeout = 2000,
    })
  end,
})
```

I just want to set the timeout for yank highlight to a higher value, but this is having no effect.

Thanks

**Comment by dpetka2001** [comment, marked as answer]
Do you have the extra for `yanky` installed? If yes, then by default it configures an autocmd for `TextYankPost` event which must be overriding Lazyvim's default. Could you try
```lua
  {
    "gbprod/yanky.nvim",
    opts = {
      highlight = {
        timer = 2000,
      },
    },
  },
```
to test if it's working for you?

**Comment by rochacbruno** [comment]
Thanks! that was the issue
